tools/server/tests/unit/test_ctx_shift.py

```diff
@@ -26,6 +26,7 @@ def test_ctx_shift_enabled():
     # 64 tokens are generated thanks to shifting the context when it gets full
     global server
     server.enable_ctx_shift = True
+    server.fa = "off"  # FIXME prompt_n assert fails otherwise
```

Comment:
@ggerganov remove this line or set it to `"on"`, then run the unit test. Alternatively, edit the unit test on master to run with FA.